brave/brave-core
# Adds tip info to confirmation dialog

Fixes: https://github.com/brave/brave-browser/issues/3023

UI implementation: https://github.com/brave/brave-ui/pull/353

Non-Recurring confirmation:
<img width="475" alt="screen shot 2019-01-23 at 11 36 24 pm" src="https://user-images.githubusercontent.com/8732757/51659448-baf19d80-1f67-11e9-9274-47908c817621.png">

Recurring confirmation:
<img width="475" alt="screen shot 2019-01-23 at 11 37 06 pm" src="https://user-images.githubusercontent.com/8732757/51659465-d066c780-1f67-11e9-95fe-e837b094a673.png">


## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:
1. Start Brave from a fresh profile, enable Rewards, accept a grant
2. Make a recurring tip to a site
3. Confirm that the confirmation overlay displays info per design
4. Make a non-recurring tip to a site
5. Confirm that the confirmation overlay displays info per design
6. Do this a few times staggering different domains and ensure info is accurate

## Reviewer Checklist:
